photon-core/src/main/java/org/photonvision/jni/PhotonJNICommon.java

```diff
@@ -69,7 +69,8 @@ protected static synchronized void forceLoad(
                 logger.error("Couldn't load shared object " + libraryName, e);
                 e.printStackTrace();
                 // logger.error(System.getProperty("java.library.path"));
-                break;
+                instance.setLoaded(false);
+                return;
```

Comment:
It looks like is loaded defaults to false already, good to be explicit I suppose. The return is the actual fix here

Reply:
Just thought I'd replicate syntax from [line 48](https://github.com/PhotonVision/photonvision/blob/82ded94bc47bb68a3e774a6e0723e54160b5d183/photon-core/src/main/java/org/photonvision/jni/PhotonJNICommon.java#L48)

Reply:
Yeah agreed